attunement.lic

```diff
@@ -24,9 +25,13 @@ class Attunement
       return
     end
 
-    room_list = if @stationary_skills_only || DRStats.necromancer?
+    room_list = if @stationary_skills_only || (DRStats.necromancer? and !@force_power_walk)
```

Comment:
A waggle set is a good idea.  I'm also a fan of a more generic name since moon mages could technically use RF/SOV as well.

ROC/ROG is not good enough for powerwalk because the main danger comes from players ACCUSING the necromancer once they have been outed and/or are known. 

There is really no downside to using EOTB as far as I know. I don't see why anyone would want to turn it off if they have it. I'd also hate to make an edge case for not having EOTB when literally every necromancer will have it.